opds_springer/book_saver.py

```diff
@@ -70,17 +86,18 @@ def parse_kbart_tsv(self, kbart_file):
         Yields:
             dict: row data
         """
-        with open(kbart_file, mode="r") as tsv:
-            tsv_reader = DictReader(tsv, dialect="excel-tab")
+        with open(self.kbart_file, mode="r") as tsv:
+            tsv_reader = DictReader(tsv, delimiter="\t")
```

Comment:
was this a character encoding issue? I learned some hard lessons about Excel, TSV and KBART this week.

Reply:
What I had thought was an issue with the code was an issue in the fixture, but I _think_ this is more accurate. I'm interested in what those lessons were!